antenna/throttler.py

```diff
@@ -332,6 +361,33 @@ def match_unsupported_windows(throttler, data):
 ]
 
 
+#: This accepts crash reports for all product packagenames
+ALL_PRODUCT_PACKAGENAMES = {}
+
+
+# Supported packagenames values; these have to match the Android_PackageName of the
+# incoming crash report
+MOZILLA_PRODUCT_PACKAGENAMES = {
```

Comment:
I'm wondering whether we'd prefer to have this data in a configuration file rather than in the source code, but I guess either way is fine.